components/molecules/ContributionsEvolution/contributions-evolution-card.tsx

```diff
@@ -58,6 +60,9 @@ interface Props {
   data: ContributionStat[];
 }
 
+// Use dynamic() to import BarDatum & ResponsiveBar from @nivo/bar
+const ResponsiveBar = dynamic(() => import("@nivo/bar").then((module) => module.ResponsiveBar));
```

Comment:
Had to make this change to as this component is ESM only. If it's not dynamically imported, you'll get the following error.

`Error: require() of ES Module /Users/nicktaylor/dev/work/app/node_modules/@nivo/bar/node_modules/d3-color/src/index.js from /Users/nicktaylor/dev/work/app/node_modules/@nivo/bar/node_modules/@nivo/colors/dist/nivo-colors.cjs.js not supported.
Instead change the require of index.js in /Users/nicktaylor/dev/work/app/node_modules/@nivo/bar/node_modules/@nivo/colors/dist/nivo-colors.cjs.js to a dynamic import() which is available in all CommonJS modules.
`

Leveraging `import dynamic from "next/dynamic";` to do this.

Related https://github.com/plouc/nivo/issues/2310